commands/command_pointer.go

```diff
@@ -79,17 +82,38 @@ func pointerCommand(cmd *cobra.Command, args []string) {
 			os.Exit(1)
 		}
 
-		oidHash := sha256.New()
-		size, err := io.Copy(oidHash, buildFile)
+		var ptr *lfs.Pointer
+		if pointerNoExtensions {
```

Comment:
As noted below, let's change this to the following, which allows us to leave our existing tests as-is and drop the `setupRepository()` call in the `else` block of this conditional statement.

```suggestion
		if pointerNoExtensions || !cfg.InRepo() {
```